src/app/home/components/CardStatusComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import theme from '../../../theme';
+import StatusIcon from '../../common/components/StatusIcon';
+import { css } from '@emotion/core';
+import {
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+
+const CardStatusComponent = ({ type, dataList, ...props }) => {
+  const successList = dataList.filter((item) => item.status === 'success');
+  const failureList = dataList.filter((item) => item.status !== 'success');
+  return (
+    <React.Fragment>
+      <Grid>
+        <GridItem span={1} style={{ textAlign: "center", margin: 'auto' }}>
```

Comment:
need to decide on how we are going about inline styles. syled div wrapper from emotion, css prop from emotion, or style prop